test-poc/base/src/test/java/org/keycloak/test/base/PagesTest.java

```diff
@@ -16,20 +16,11 @@ public class PagesTest {
     LoginPage loginPage;
 
     @Test
-    public void testLoginFromWelcome() {
+    public void testLoginFromWelcome() throws InterruptedException {
         welcomePage.navigateTo();
-        try {
-            Thread.sleep(1000);
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
-        }
+        Thread.sleep(1000);
```

Comment:
Once Chrome and Firefox options are in place this can be removed.

Reply:
Removed